db/schema.rb

```diff
@@ -16,8 +16,8 @@
   create_table "batch_invitation_application_permissions", force: :cascade do |t|
     t.integer  "batch_invitation_id",     limit: 4, null: false
     t.integer  "supported_permission_id", limit: 4, null: false
-    t.datetime "created_at",                        null: false
-    t.datetime "updated_at",                        null: false
+    t.datetime "created_at"
```

Comment:
@benilovj I'm not sure why but running migrations from version 0 gave me this schema :confused: . But I'll run `schema:load` instead